### PR TITLE
Use command to run replicant redmine until we upgrade

### PR DESCRIPTION
**Pull request by fahlmant**
Unicorn requires a newer version of ruby, so chaning to this in the interim until we upgrade the redmine.

### CODE REVIEW

File: recipes/app2.rb [hunk 1]
```diff
@@ -192,7 +192,6 @@
     user 'replicant'
     working_directory '/home/replicant/redmine'
     pid_file '/home/replicant/pids/unicorn.pid'
-    exec_start '/home/replicant/.rvm/bin/rvm 2.3.0 do bundle exec '\
-      'unicorn -l 8090 -c unicorn.rb -E deployment -D'
+    exec_start 'RAILS_ENV=production ./script/server -b 140.211.9.86 -p 8090'
```
Comment (ramereth) [inline recipes/app2.rb]: Don't we need to invoke rvm so that it uses the proper ruby environment?

Comment (fahlmant) [inline recipes/app2.rb]: @ramereth I'm not sure. This is the command that's running in the screen session, and it seems to choose 1.8.7

Comment (ramereth) [inline recipes/app2.rb]: I recommend you add ``/home/replicant/.rvm/bin/rvm 2.3.0 do bundle exec`` to before the ``./script/server`` command to ensure it uses the correct rubies.

Comment (fahlmant) [inline recipes/app2.rb]: @ramereth We are trying to use 1.8.7, not 2.3.0

Comment (ramereth) [inline recipes/app2.rb]: Right, choose whatever version you need but you need to go through rvm regardless.

Comment (fahlmant) [inline recipes/app2.rb]: Bundle needs a newer version of ruby than redmine is running with

Comment (ramereth) [inline recipes/app2.rb]: Can you at least run it without bundle exec? Does that work?

**Comment by fahlmant** [comment]
@ramereth Updated the command.

**Comment by fahlmant** [comment]
!bump patch ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v1.11.3.  Have a nice day!

**Pull request merged**